packages/genesys-spark-components/src/components/legacy/gux-advanced-dropdown-legacy/gux-advanced-dropdown.scss

```diff
@@ -43,7 +45,8 @@ gux-popup {
 
       &:focus-visible {
         outline: none;
```

Comment:
Is this style still needed ?  I think its overridden by the `focus.gux-focus-ring`